config/nexmo.php

```diff
@@ -27,6 +27,7 @@
     */
 
     'signature_secret' => function_exists('env') ? env('NEXMO_SIGNATURE_SECRET', '') : '',
+    'signature_method' => function_exists('env') ? env('NEXMO_SIGNATURE_METHOD', '') : '',
```

Comment:
(More of a question) Should we not default this to a specific hashing method? [The docs say that we default to MD5](https://developer.nexmo.com/concepts/guides/signing-messages).

Reply:
We do default to MD5, but there are discussions around deprecating `md5` and `sha1` and making `sha256` the default for new accounts. With that in mind, I'd like our consumers to be explicit in the hashing method they'd like to use